packages/cozy-codemods/src/transforms/replace-icons-svgr.js

```diff
@@ -70,5 +70,36 @@ export default function replaceSvgrIcons(file, api) {
       }
     })
 
+  root
+    .find(j.JSXOpeningElement, {
+      name: {
+        name: 'Avatar'
+      }
+    })
+    .forEach(path => {
+      const iconAttr = path.node.attributes.find(
+        attr => attr.name && attr.name.name === 'icon'
+      )
+
+      if (iconAttr && iconAttr.value.type === 'Literal') {
+        const componentName = iconNameToComponentName(iconAttr.value.value)
+
+        imports.ensure(
+          root,
+          {
+            default: `${componentName}Icon`
+          },
+          `cozy-ui/transpiled/react/Icons/${componentName}`
+        )
+        imports.ensure(
+          root,
+          {
+            default: `Icon`
+          },
+          `cozy-ui/transpiled/react/Icon`
+        )
+        iconAttr.value = `{<Icon icon={${componentName}Icon} />}`
+      }
+    })
```

Comment:
Est-ce que ce ne serait pas intéressant de mettre une petite doc / commentaire pour expliquer pourquoi on fait ça ?

Reply:
A doc for who ?

Reply:
J'avais pas vu qu'on était dans `cozy-codemods` :)